main.cpp

```diff
@@ -55,13 +59,38 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
-}
-
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+
+    int rainDrops = 45;    
+    int cows = 100;
+    int planets = 8;
+
+    float numSamplesToRender = 56000.0f;
+    float decibelsToGain = 0.5f;
+    float ageOfMySonInYears = 3.5f;
+
+    double lightYearsToAlienLife = 123670978.454;
+    double slicesOfCakeLeftInMyHouse = 0.0002;
+    double percentageOfWheelsOnBrokenCar = 0.75;
+
+    bool rainToday = true;
+    bool rainedLastFriday = false;
+    bool rainTomorrow = 1;
```

Comment:
never use '0' or '1' to represent a boolean.  always aim to make your code as readable as possible
- [ ] revise